bundles/binding/org.openhab.binding.mios/src/main/java/org/openhab/binding/mios/internal/config/ServiceAliases.properties

```diff
@@ -109,3 +109,4 @@ Tuning1=urn:micasaverde-com:serviceId:Tuning1
 VideoAdjustment1=urn:micasaverde-com:serviceId:VideoAdjustment1
 Volume1=urn:micasaverde-com:serviceId:Volume1
 WMC1=urn:micasaverde-com:serviceId:WMC1
+RGBController1=urn:upnp-org:serviceId:RGBController1
```

Comment:
Ok, a hyperlink that in the doc (see below) should at least cover it.

Double-check the GIT diff, it looks like it's got an issue with the EOL or EOF char.